webapp/src/main/java/com/plhal/ares/webapp/AresApiConfig.java

```diff
@@ -1,38 +1,52 @@
 package com.plhal.ares.webapp;
 
-import com.plhal.ares.model.DataRepository;
-import com.plhal.ares.model.DataRepositoryImpl;
-import com.plhal.ares.model.DataRepositoryProperties;
+import com.plhal.ares.parser.*;
 import com.plhal.ares.service.DataService;
 import com.plhal.ares.service.DataServiceImpl;
+import org.flywaydb.core.Flyway;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+import javax.sql.DataSource;
 
 @Configuration
+@Import(DbConfig.class)
 public class AresApiConfig {
 
+    @Bean
+    public Flyway flyway(DataSource dataSource) {
+        Flyway flyway = Flyway.configure()
+                .dataSource(dataSource)
+                .schemas("ares")
```

Comment:
Jak jste psal, tak schema nemuze byt hardcodovano v kodu. Plati to v podstate o temer kaze "magicke" konstante. K tomu slouzi prinejhorsim konfiguraci. K cemu slouzi `spring.flyway.default-schema` a `spring.flyway.locations`. Vice o properties v [dokumentaci](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html).